transport-native-kqueue/pom.xml

```diff
@@ -79,6 +79,8 @@
                   <configureArgs>
                     <arg>${jni.compiler.args.ldflags}</arg>
                     <arg>${jni.compiler.args.cflags}</arg>
+                    <!-- weak_import was added in 10.2 so ensure we explicitly set the artifact -->
+                    <arg>MACOSX_DEPLOYMENT_TARGET=10.2</arg>
```

Comment:
Sure. The artifact failed to link on my 10.10 machine because the weak linking feature was not enabled. This meant that accept4 (which is defined as `__attribute__((weak_import))`) failed to link (which is expected) but the linker threw an error.